README.md

```diff
@@ -88,6 +91,9 @@ productTypeImport.importProductType(productType)
   // }
 })
 ```
+
+When there is an error during processing productTypes, process will by default save this error to `summary.errors` array and reject. If the configuration flag `importerConfig.rejectOnError` is set to `false` the importer will only push error to summary and then continue with next productType.
```

Comment:
To a normal user; do we want to explain error rejection? Could this be somewhat similar to `continueOnProblems`?

Reply:
This module does not have any CLI interface and has to be imported to JS project using `require` or `import` so the user of this module should have some technical knowledge.

Reply:
Also, I changed the name of flag to proposed `continueOnProblems` because it is used in other repos.